### PR TITLE
Fix #4172 - name of firewall cpl changed

### LINKED ISSUE #4172: Windows Firewall example
Example 2 should be revised to either "Windows Defender Firewall" or "Microsoft.WindowsFirewall" for -Name or -CanonicalName when getting the ControlPanelItem.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: d4622146-2027-5913-370a-3a0a182b2c7d
* Version Independent ID: 6f0d5c88-4d91-e080-5f21-c6876c6c2fc6
* Content: [Show-ControlPanelItem (Microsoft.PowerShell.Management)](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.management/show-controlpanelitem?view=powershell-5.1#feedback)
* Content Source: [reference/5.1/Microsoft.PowerShell.Management/Show-ControlPanelItem.md](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/5.1/Microsoft.PowerShell.Management/Show-ControlPanelItem.md)
* Product: **powershell**
* Technology: **powershell-cmdlets**
* GitHub Login: @daxian-dbw
* Microsoft Alias: **dongbow**

### PR DESCRIPTION
**Pull request by sdwheeler**
Version(s) of document impacted
------------------------------
- [ ] Impacts 6.next document
- [ ] Impacts 6 document
- [x] Impacts 5.1 document
- [x] Impacts 5.0 document
- [x] Impacts 4.0 document
- [x] Impacts 3.0 document

Reason(s) for not updating all version of documents
--------------------------------------------------
- [ ] The documented feature was introduced in version (list version here) of PowerShell
- [ ] This issue only shows up in version (list version(s) here) of the document
- [ ] This PR partially fixes the issue, and issue #<insert here> tracks the remaining work

### CODE REVIEW

**Comment by zjalexander** [comment]
Docs Build status updates of commit _[9f32c31](https://github.com/sdwheeler/PowerShell-Docs/pull/4196/commits/9f32c31e689650b4b3c4269317969567f32a2ecb)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/3.0/Microsoft.PowerShell.Management/Show-ControlPanelItem.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue4172/reference/3.0/Microsoft.PowerShell.Management/Show-ControlPanelItem.md) | :white_check_mark:Succeeded | |
[reference/4.0/Microsoft.PowerShell.Management/Show-ControlPanelItem.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue4172/reference/4.0/Microsoft.PowerShell.Management/Show-ControlPanelItem.md) | :white_check_mark:Succeeded | |
[reference/5.0/Microsoft.PowerShell.Management/Show-ControlPanelItem.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue4172/reference/5.0/Microsoft.PowerShell.Management/Show-ControlPanelItem.md) | :white_check_mark:Succeeded | |
[reference/5.1/Microsoft.PowerShell.Management/Show-ControlPanelItem.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue4172/reference/5.1/Microsoft.PowerShell.Management/Show-ControlPanelItem.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C22%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201904222131491915-4196%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=njqIx%2FqDxdUHiSW45pNxxd%2BlI2YfK8VKL5rCXJGP1B0%3D&st=2019-04-22T21%3A31%3A24Z&se=2019-05-23T21%3A36%3A24Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: reference/3.0/Microsoft.PowerShell.Management/Show-ControlPanelItem.md [hunk 1]
```diff
@@ -1,74 +1,96 @@
...
-PS C:\> Get-ControlPanelItem -Name "Windows Firewall" | Show-ControlPanelItem
+
+### Example 2: Pipe a control panel item to this cmdlet
+
+This example opens the **Windows Defender Firewall** control panel item on the local computer.
+The name of the Windows Firewall control panel item has changed over the versions of Windows. This
+example uses a wildcard pattern to find the control panel item.
+
+```powershell
+Get-ControlPanelItem -Name "*Firewall" | `Show-ControlPanelItem`
```
Comment (DCtheGeek) [inline reference/3.0/Microsoft.PowerShell.Management/Show-ControlPanelItem.md]: Don't think the codefences are intended to remain around **Show-ControlPanelItem** inside this codefence.

**Comment by zjalexander** [comment]
Docs Build status updates of commit _[418caf5](https://github.com/sdwheeler/PowerShell-Docs/pull/4196/commits/418caf5b8f80a099784b1f2ec384c65217d40822)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/3.0/Microsoft.PowerShell.Management/Show-ControlPanelItem.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue4172/reference/3.0/Microsoft.PowerShell.Management/Show-ControlPanelItem.md) | :white_check_mark:Succeeded | |
[reference/4.0/Microsoft.PowerShell.Management/Show-ControlPanelItem.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue4172/reference/4.0/Microsoft.PowerShell.Management/Show-ControlPanelItem.md) | :white_check_mark:Succeeded | |
[reference/5.0/Microsoft.PowerShell.Management/Show-ControlPanelItem.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue4172/reference/5.0/Microsoft.PowerShell.Management/Show-ControlPanelItem.md) | :white_check_mark:Succeeded | |
[reference/5.1/Microsoft.PowerShell.Management/Show-ControlPanelItem.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue4172/reference/5.1/Microsoft.PowerShell.Management/Show-ControlPanelItem.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C22%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201904222134521848-4196%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=U4UUqYhTAy6INl6lI7FkNxd2vprnZr0YGsZ1y2loehw%3D&st=2019-04-22T21%3A36%3A21Z&se=2019-05-23T21%3A41%3A21Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: reference/3.0/Microsoft.PowerShell.Management/Show-ControlPanelItem.md (hunk 1 above)
Comment (sdwheeler) [reply reference/3.0/Microsoft.PowerShell.Management/Show-ControlPanelItem.md]: Fixed

**Comment by zjalexander** [comment]
Docs Build status updates of commit _[1d6776d](https://github.com/sdwheeler/PowerShell-Docs/pull/4196/commits/1d6776d99f03f754441c66e3623b7b156f059798)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/3.0/Microsoft.PowerShell.Management/Show-ControlPanelItem.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue4172/reference/3.0/Microsoft.PowerShell.Management/Show-ControlPanelItem.md) | :white_check_mark:Succeeded | |
[reference/4.0/Microsoft.PowerShell.Management/Show-ControlPanelItem.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue4172/reference/4.0/Microsoft.PowerShell.Management/Show-ControlPanelItem.md) | :white_check_mark:Succeeded | |
[reference/5.0/Microsoft.PowerShell.Management/Show-ControlPanelItem.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue4172/reference/5.0/Microsoft.PowerShell.Management/Show-ControlPanelItem.md) | :white_check_mark:Succeeded | |
[reference/5.1/Microsoft.PowerShell.Management/Show-ControlPanelItem.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue4172/reference/5.1/Microsoft.PowerShell.Management/Show-ControlPanelItem.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C22%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201904222142439965-4196%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=GWa0VfH8Llgy0Sb1S%2FCcWJg8Wgmcb7SYci6WI%2FVj5dk%3D&st=2019-04-22T21%3A43%3A06Z&se=2019-05-23T21%3A48%3A06Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by zjalexander** [comment]
Docs Build status updates of commit _[1d6776d](https://github.com/sdwheeler/PowerShell-Docs/pull/4196/commits/1d6776d99f03f754441c66e3623b7b156f059798)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/3.0/Microsoft.PowerShell.Management/Show-ControlPanelItem.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue4172/reference/3.0/Microsoft.PowerShell.Management/Show-ControlPanelItem.md) | :white_check_mark:Succeeded | |
[reference/4.0/Microsoft.PowerShell.Management/Show-ControlPanelItem.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue4172/reference/4.0/Microsoft.PowerShell.Management/Show-ControlPanelItem.md) | :white_check_mark:Succeeded | |
[reference/5.0/Microsoft.PowerShell.Management/Show-ControlPanelItem.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue4172/reference/5.0/Microsoft.PowerShell.Management/Show-ControlPanelItem.md) | :white_check_mark:Succeeded | |
[reference/5.1/Microsoft.PowerShell.Management/Show-ControlPanelItem.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue4172/reference/5.1/Microsoft.PowerShell.Management/Show-ControlPanelItem.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C22%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201904222151418304-4196%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=zsIBO16OMJZWtFGxbxRFBH4YGWUjsn%2BK9uOOjuEbFq0%3D&st=2019-04-22T21%3A51%3A36Z&se=2019-05-23T21%3A56%3A36Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**